feg/cloud/go/services/basic_acct/servicers/southbound/service.go

```diff
@@ -25,6 +25,7 @@ import (
 	"magma/feg/cloud/go/feg"
 	"magma/feg/cloud/go/protos"
 	"magma/feg/cloud/go/services/basic_acct"
+	"magma/feg/cloud/go/services/basic_acct/servicers"
```

Comment:
Let's just move all the basic_acct servicers in 1 PR (?)

Reply:
What do you mean by all basic_acct servicers ?

Reply:
I believe we can just move the referenced file, rather than importing it

Reply:
Done